test/integration/driver.go

```diff
@@ -48,8 +48,8 @@ const (
 	// budget is the maximum number of retries with 1s delays
 	budget = 90
 
-	// Mixer SHA *update manually*
-	mixerSHA = "ea3a8d3e2feb9f06256f92cda5194cc1ea6b599e"
+	// Mixer tag *update manually*
+	mixerTag = "demo"
```

Comment:
Unfortunately, manual tags like "demo" will not work if the image is stored in docker hub. Docker hub does not support manually tagging an image pushed by the build system. I need to use the same tag as pushed by the build system on both gcr.io and docker hub, and this is the short SHA.